wiki/Installing-Oppia-Android.md

```diff
@@ -150,8 +526,95 @@ We strongly recommend running tests on Robolectric which is faster because it do
 
 - To learn about how we write tests at oppia-android, please refer to: [Oppia Android Testing](https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing).
 
-- To get started with Bazel, please see [Oppia-Bazel-Setup-Instructions](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Setup-Instructions) and follow the instructions appropriate to your OS.
-
-- If you run into any issues during your setup, search our [discussions](https://github.com/oppia/oppia-android/discussions) forum or start a new discussion.
-
-- Please see  [Troubleshooting-Installation](https://github.com/oppia/oppia-android/wiki/Troubleshooting-Installation) for solutions to some common setup issues.
+- To learn more about getting started with Bazel, please see [Oppia-Bazel-Glossary](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Glossary).
+
+### Troubleshooting Installation
+#### General issues
+1. If the Static Checks on your PR keeps failing, please refer to [instructions on running static checks locally](https://github.com/oppia/oppia-android/wiki/Static-Analysis-Checks#how-to-run-static-checks-locally) in order to catch these errors before pushing.
+2. If you are using Android Studio or another UI-based git to push, you might see unclear errors such as `Error: Failed to push some refs to git@github.com:<your_user_name>/oppia-android.git`. We recommend using the command line to run all git commands.
+
+#### Bazel issues
+The team is in the process of migrating away from Gradle to Bazel, so while the project still has Gradle build files, they are no longer maintained and cannot be used to build the project.
+
+Please note that:
+- It's currently recommended that all team members use **Bazel** for their active development in Android Studio. While some team members use the Bazel Android Studio plugin, day-to-day Bazel development currently suffers from:
+    - Significant memory overhead that continues to grow without careful pruning (i.e. periodic shutdowns of the local Bazel build server). On some Linux distros or MacOS, this can result in a Kernel panic when memory is fully exhausted.
+    - Various symbolic errors throughout the codebase that can make it much more difficult to jump to specific symbols (though, unlike Gradle, all code including scripts are editable and runnable within Android Studio).
+    - Syntax highlighting errors in some instances.
+- As the team finishes the migration to Bazel, communications and instructions will be sent ahead of time, and we will continually update our documentation with the latest changes.
+
+1. No matching toolchains (sdk_toolchain_type)
```

Comment:
```suggestion

Other problems that you may run into:

1. No matching toolchains (sdk_toolchain_type)
```

(to provide some spacing between the 'please note that' and the numeration of problems).

Reply:
Done